packages/vscode/src/plugins/nextjs/tests/__helpers__/buildVariations.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  export_variations,
+  import_variations,
+  page_export_variations,
+} from '../__fixtures__/variations'
+type KeyOf = keyof typeof import_variations
+type KeyOfo = keyof typeof export_variations
+type KeyOfoo = keyof typeof page_export_variations
```

Comment:
Maybe refactor these names to something more explicit.

Reply:
Forgot about those, sure thing